<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BillSplittr</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(236, 236, 236);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>-1</number>
        </property>
        <item>
         <widget class="QLabel" name="numPeople_label">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#0f4c75;&quot;&gt;# of People&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="numPeople">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 36px;
font: bold;
color: #0f4c75;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt; font-weight:600; color:#0f4c75;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>-1</number>
        </property>
        <item>
         <widget class="QLabel" name="total_label">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#0f4c75;&quot;&gt;Total&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="totalPrice">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 36px;
font: bold;
color: #0f4c75;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt; font-weight:600; color:#0f4c75;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>430</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabWidget QWidget
{
	background-color: #dae1e7;
	border-radius: 10px;
}

QTabWidget::pane { 
	border: none;
	top:15px; 
}

QTabBar::tab {
	width: 180px;
	height: 40px;
	padding: 5px;
	font: 18px;
	font-weight: bold;
	color: #5e5e5e;
	background: rgb(236, 236, 236);
	border: outset;
	border-radius: 1px;
	border-width: 3px;
	border-top: none;
	border-right: none;
	border-left: none;
	border-bottom-color: #bbbbbb;
}

QTabBar::tab:selected {
	border: outset;
	color: #0f4c75;
	border-width: 3px;
	border-top: none;
	border-right: none;
	border-left: none;
	border-bottom-color: #3282b8;
}</string>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="list_tab">
       <attribute name="title">
        <string>☰ List</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="listName_label">
            <property name="maximumSize">
             <size>
              <width>133</width>
              <height>100</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 16px;
font: bold;
color: #3282b8;</string>
            </property>
            <property name="text">
             <string>List Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="price_label">
            <property name="maximumSize">
             <size>
              <width>133</width>
              <height>100</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 16px;
font: bold;
color: #3282b8;</string>
            </property>
            <property name="text">
             <string>Price</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="perPerson_label">
            <property name="maximumSize">
             <size>
              <width>133</width>
              <height>100</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 16px;
font: bold;
color: #3282b8;</string>
            </property>
            <property name="text">
             <string>Per Person</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QScrollArea" name="list_scrollArea">
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>200</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 16px;
font-weight: bold;
color: #0f4c75;</string>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="list_scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>336</width>
             <height>200</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="listInput">
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #f6f5f5;
border-radius: 4px;
color: #0f4c75;
font: 14px;
font: bold;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="listAdd_button">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	font: 14px;
	font: bold;
	color: #dae1e7;
	border: solid;
	border-radius: 4px;
	background-color: #3282b8;
}

QPushButton:pressed{
	background-color: #0f4c75;
}</string>
            </property>
            <property name="text">
             <string>Add ⏎</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="clearAllList_button">
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	font: 16px;
	font: bold;
	text-align: left;
	color: #3282b8;
	border: none;
	background: none;
	background-color: transparent;
}

QPushButton:pressed{
	color: #0f4c75;
}</string>
          </property>
          <property name="text">
           <string>Clear All List ⌫</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="payer_tab">
       <attribute name="title">
        <string>웃 Payer</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="name_label">
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>100</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 16px;
font: bold;
color: #3282b8;</string>
            </property>
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="pay_label">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>100</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 16px;
font: bold;
color: #3282b8;</string>
            </property>
            <property name="text">
             <string>Pay   </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QScrollArea" name="name_scrollArea">
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>200</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 16px;
font-weight: bold;
color: #0f4c75;</string>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <widget class="QWidget" name="name_scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>336</width>
             <height>200</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLineEdit" name="nameInput">
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #f6f5f5;
border-radius: 4px;
color: #0f4c75;
font: 14px;
font: bold;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="nameAdd_button">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	font: 14px;
	font: bold;
	color: #dae1e7;
	border: solid;
	border-radius: 4px;
	background-color: #3282b8;
}

QPushButton:pressed{
	background-color: #0f4c75;
}</string>
            </property>
            <property name="text">
             <string>Add ⏎</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="clearAllName_button">
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	font: 16px;
	font: bold;
	text-align: left;
	color: #3282b8;
	border: none;
	background: none;
	background-color: transparent;
}

QPushButton:pressed{
	color: #0f4c75;
}</string>
          </property>
          <property name="text">
           <string>Clear All Name ⌫</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="createBill_button">
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {	
	font: 14px;
	font: bold;
	color: #dae1e7; 
	border: outset;
	border-radius: 10px;
	background-color: #3282b8;
}

QPushButton:pressed{
	background-color: #0f4c75;
}</string>
      </property>
      <property name="text">
       <string>Create Bill ⏎</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>nameInput</sender>
   <signal>returnPressed()</signal>
   <receiver>nameAdd_button</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>470</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listInput</sender>
   <signal>returnPressed()</signal>
   <receiver>listAdd_button</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>475</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>476</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
